### ISSUE TITLE
Form2Task : show pending answers badge somewhere

### ISSUE THREAD

**Issue by seddik** [gitscrum-team/GitScrum-CE#7]
Form2Task is an excellent feature, but when someone create a new request, we get a notification email only, in the gitscrum app you can't find out about them unless you go there.

I suggest adding a new small badge on the sidebar to notify users to convert those answers to tasks.

something simple like this
![image](https://user-images.githubusercontent.com/6674347/82891716-1e1d8680-9f46-11ea-8d75-f9f37f6f280e.png)

**Comment by seddik** [comment]
@renatomarinho  I am amost done with this feature, I just need an **EndPoint** showing **unconverted** items from **Form2Tasks** only.

example : 
`https://api.gitscrum.com/form-answers?company_slug=xxx&project_slug=xxx&status=0`.


I don't want to loop over all pages then check status=0, then set the badge.

**Comment by renatomarinho** [comment]
Yes. It's a great improvement. We are going to add this to the API.
